openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -12,11 +23,13 @@
     "start_time",
     "end_time",
     "url_name",
+    "template_for_committee_id",
```

Comment:
This is not a setting but a relation, I would not include it here but list it separately

Reply:
Added separatly.